modules/500-operator-trivy/charts/helm_lib

```diff
@@ -0,0 +1 @@
+/deckhouse/helm_lib
```

Comment:
Wrong symlink. Should be `/deckhouse/helm_lib/charts/deckhouse_lib_helm`

Reply:
Fixed.